scripts/convert_animatediff_motion_lora_to_diffusers.py

```diff
@@ -51,4 +59,12 @@ def get_args():
             continue
         output_dict.update({f"unet.{module_name}": params})
 
-    save_file(output_dict, f"{args.output_path}/diffusion_pytorch_model.safetensors")
+    os.makedirs(args.output_path, exist_ok=True)
+
+    filepath = os.path.join(args.output_path, "diffusion_pytorch_model.safetensors")
+    save_file(output_dict, filepath)
+
+    if args.push_to_hub:
+        api = HfApi()
+        repo_id = api.create_repo(args.output_path, exist_ok=True).repo_id
+        api.upload_folder(repo_id=repo_id, folder_path=args.output_path, repo_type="model")
```

Comment:
We can directly use `create_repo()` and `upload_folder()` from `huggingface_hub`. We don't need to use `HfApi()` here.